cedar-policy/src/api.rs

```diff
@@ -4524,4 +4537,34 @@ mod test {
                 )
         );
     }
+
+    #[test]
+    fn into_iter_entities() {
+        let test_data = r#"
+        [
+        {
+        "uid": {"type":"User","id":"alice"},
+        "attrs": {
+            "age":19,
+            "ip_addr":{"__extn":{"fn":"ip", "arg":"10.0.1.101"}}
+        },
+        "parents": [{"type":"Group","id":"admin"}]
+        },
+        {
+        "uid": {"type":"Group","id":"admin"},
+        "attrs": {},
+        "parents": []
+        }
+        ]
+        "#;
+
+        let list = Entities::from_json_str(test_data, None).unwrap();
+        let mut list_out: Vec<String> = Vec::new();
```

Comment:
```suggestion
        let mut list_out: Vec<String> = list.into_iter().map(|entity| entity.uid().id().to_string()).collect();
```
Tiny nit, but slightly more idiomatic

Reply:
done. thanks